recipes-browser/wpebackend-fdo/wpebackend-fdo.inc

```diff
@@ -9,5 +9,3 @@ PROVIDES += "virtual/wpebackend"
 RPROVIDES_${PN} += "virtual/wpebackend"
 
 inherit cmake
-
-FILES_${PN} += "${libdir}/libWPEBackend-fdo-0.1.so"
```

Comment:
Shouldn't this be moved to  the wpebackend-fdo_0.1.bb recipe

Reply:
It's removed. Not sure what you mean :)

Reply:
I have no idea why, the symlink is not included in the package containing the libraries (`so` files) but instead is part of the wpebackend-fdo-dev package... I'll leave resolving this issue to an actual Yocto expert and include the -dev pkg in my image meanwhile.

Reply:
I have no idea why, the symlink is not included in the package containing the libraries (so files) but instead is part of the wpebackend-fdo-dev package... I'll leave resolving this issue to an actual Yocto expert and include the -dev pkg in my image meanwhile.

Reply:
working in rocko at least, just with this change:

```
+FILES_SOLIBSDEV = ""
+FILES_${PN} += "${libdir}/libWPEBackend-default.so ${libdir}/libWPEBackend-fdo-0.1.so"
+INSANE_SKIP_${PN} ="dev-so"
+
+do_install_append () {
+    install -d "${D}/usr/lib"
+    ln -s libWPEBackend-fdo-0.1.so "${D}/usr/lib/libWPEBackend-default.so"
+}
```


`FILES_SOLIBSDEV=""` forces all libraries to be included in the RPM package

Reply:
Sent a PR for this https://github.com/Igalia/meta-webkit/pull/37